includes/apple-exporter/builders/class-builder.php

```diff
@@ -48,112 +68,110 @@ public function to_array() {
 	/**
 	 * Builds the content.
 	 *
-	 * @abstract
 	 * @access protected
 	 */
 	protected abstract function build();
 
-	// Isolate dependencies
-	// ------------------------------------------------------------------------
-
 	/**
-	 * Gets the content ID.
+	 * Gets the content byline.
 	 *
 	 * @access protected
-	 * @return mixed
+	 * @return string The byline from the content object.
 	 */
-	protected function content_id() {
-		return $this->content->id();
+	protected function content_byline() {
+		return $this->content->byline();
 	}
 
 	/**
-	 * Gets the content title.
+	 * Gets the content cover.
 	 *
 	 * @access protected
-	 * @return string
+	 * @return string The URL for the cover image from the content object.
 	 */
-	protected function content_title() {
-		return $this->content->title() ?: __( 'Untitled Article', 'apple-news' );
+	protected function content_cover() {
+		return $this->content->cover();
 	}
 
 	/**
-	 * Gets the content body.
+	 * Gets the content ID.
 	 *
 	 * @access protected
-	 * @return string
+	 * @return int The ID from the content object.
 	 */
-	protected function content_text() {
-		return $this->content->content();
+	protected function content_id() {
+		return $this->content->id();
 	}
 
 	/**
 	 * Gets the content intro.
 	 *
 	 * @access protected
-	 * @return Intro
+	 * @return string The intro from the content object.
 	 */
 	protected function content_intro() {
 		return $this->content->intro();
 	}
 
 	/**
-	 * Gets the content cover.
+	 * Gets the content nodes.
 	 *
 	 * @access protected
-	 * @return Cover
+	 * @return array The nodes from the content object.
 	 */
-	protected function content_cover() {
-		return $this->content->cover();
+	protected function content_nodes() {
+		return $this->content->nodes();
 	}
 
 	/**
 	 * Gets a content setting.
+	 *
+	 * @param string $name The setting name to retrieve.
+	 *
 	 * @access protected
-	 * @param string $name
-	 * @return string
+	 * @return mixed The value for the setting.
 	 */
 	protected function content_setting( $name ) {
 		return $this->content->get_setting( $name );
 	}
 
 	/**
-	 * Gets the content byline.
+	 * Gets the content body.
 	 *
 	 * @access protected
-	 * @return Byline
+	 * @return string The body text from the content object.
 	 */
-	protected function content_byline() {
-		return $this->content->byline();
+	protected function content_text() {
+		return $this->content->content();
 	}
 
 	/**
-	 * Gets the content nodes.
+	 * Gets the content title.
 	 *
 	 * @access protected
-	 * @return array
+	 * @return string The title from the content object, or a fallback title.
 	 */
-	protected function content_nodes() {
-		return $this->content->nodes();
+	protected function content_title() {
+		return $this->content->title() ?: __( 'Untitled Article', 'apple-news' );
```

Comment:
Not sure this is a valid concern, but seeing that WP is backward compat to php 5.2.x, this _might_ be an issue (though highly unlikely). Support for this shorthand syntax for ternary operations was introduced in 5.3.

Not sure this is a risk, just thought I'd flag it since this is a plugin.